lib/generators/blacklight/templates/catalog_controller.rb

```diff
@@ -180,6 +180,10 @@ class <%= controller_name.classify %>Controller < ApplicationController
     # If there are more than this many search results, no spelling ("did you 
     # mean") suggestion is offered.
     config.spell_max = 5
+
+    # Configuration for autocomplete suggestor
+    config.autocomplete_enabled = true
```

Comment:
Can we just say that having a non-nil `autocomplete_path` implies that autocomplete is enabled?